opennft/matlab/nfbPreprVol/getMAT.m

```diff
@@ -49,72 +49,4 @@
 patient_to_tal   = diag([-1 -1 1 1]);
 mat              = patient_to_tal * dicom_to_patient * analyze_to_dicom;
 
-% Maybe flip the image depending on SliceNormalVector from 0029,1010
```

Comment:
Why have you removed this section?

Reply:
Because we couldn't find these fields in Siemens XA30 format and left a possibility to use it. For other Siemens formats checked so far, this case is rather unused given getMAT.m is also rarely used. I think, spatial orientation issues can be resolved given that Reference EPI, ROIs, and real-time EPIs are identically oriented. This is how we customized it for XA30. May be later this could be further generalized.

Reply:
Having the ROIs and the EPIs only help if the ROIs have been defined in the EPI images rather than on the structural one

Reply:
Well, I simply don't know yet which field to check in XA30 for this, which field to search?.. perhaps it is in another field as a few of necessary parameters that we already found and, which requires a deep investigation of XA30. Of course, this remains open to someone who provides a definite solution..

Reply:
How about disabling the section if the DICOM is in XA30 format?

Reply:
Right, this is what I was thinking about, will do it.

Reply:
Thanks!

Reply:
Actually I was sure we just commented it. Here it is back: cc1a30b6ffa61797b9efdade9655b518242db74c

Reply:
Thanks! Passing the flag rather than the whole `P` structure only might have been more efficient. I am saying this because OpenNFT is already quite memory-hungry. :)